pytest_logger/plugin.py

```diff
@@ -164,13 +187,20 @@ def _refresh_link(source, link_name):
         pass
 
 
-def _make_logsdir(tmpdirhandler, logdirlinks):
+def _make_logsdir_tmpdir(tmpdirhandler):
     logsdir = tmpdirhandler.getbasetemp()
     if logsdir.basename.startswith('popen-gw'):
         logsdir = logsdir.join('..')
     logsdir = logsdir.join('logs').ensure(dir=1)
-    for link in logdirlinks:
-        _refresh_link(str(logsdir), link)
+    return logsdir
+
+
+def _make_logsdir_dir(dstname, cleandir=True):
```

Comment:
This is ready for next extension to implement switch `disable_clean_dir` or such. If we consider those make_logdirs funtions, both could be considered as dead because used only in one place and could be placed there.